src/bootchart.c

```diff
@@ -351,6 +352,10 @@ int main(int argc, char *argv[]) {
         rlim.rlim_max = 4096;
         (void) setrlimit(RLIMIT_NOFILE, &rlim);
 
+        schfd = open("/proc/sys/kernel/sched_schedstats", O_WRONLY);
+        if (schfd)
```

Comment:
Yeah, we're probably fine not using autoclose here, I'll update this to close right away.